<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depart</class>
 <widget class="QMainWindow" name="depart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>企业人事管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>企业部门信息管理</string>
    </property>
   </widget>
   <widget class="QLabel" name="strLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>您登录于</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectDepartBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查询部门信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alterDepartBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>修改部门信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteDepartBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除部门信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addDepartBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加部门信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统设置</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="helpAaction"/>
    <addaction name="alterAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>注销</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aboutAction">
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
  <action name="helpAaction">
   <property name="text">
    <string>帮助信息</string>
   </property>
  </action>
  <action name="alterAction">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
